### ISSUE TITLE
can you add "delete" function as well

### ISSUE THREAD

**Issue by cmddot** [KB5201314/UndoForAndroid#4]
thanks.

**Comment by imlk0** [comment]
Do you mean to add a button named "delete" next to "undo" and "redo"? But why not do this via the delete button on your IME?

**Comment by cmddot** [comment]
correct. sometimes i just want to delete some characters, as you mentioned currently I have to pop up IME and then hide it. if I have this button, it's extremely useful.

**Comment by imlk0** [comment]
I am afraid that adding a "delete"  may cause pop items to be too compact. Another reason is that the project is already lacking in maintenance.
You can try using "cut" as an alternative of "delete".
